server/project/accounts/models.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Create your models here.
+
+from django.db import models
+from django.utils import timezone
+
+ACCOUNT_TYPES = (('a', 'Asset'), ('l', 'Liability'), ('e', 'Equity'))
+BOOL_TYPES = (('t', 'True'), ('f', 'False'), ('n', 'Null'))
+
+
+class Account(models.Model):
+    account_id = models.IntegerField(primary_key=True)
```

Comment:
We don't need to define this as the model will have an auto-incrementing id field by default.

Reply:
Addressed